tests/functional/oauth/test_oauth.py

```diff
@@ -75,7 +96,6 @@ def snowflake_profile(self):
         return profile
 
     @pytest.mark.skipif(OAUTH_TESTS_DISABLED, reason='oauth tests disabled')
-    @use_profile('snowflake')
-    def test_snowflake_basic(self):
-        self.run_dbt()
-        self.assertManyRelationsEqual([['MODEL_3'], ['MODEL_4']])
+    def test_snowflake_basic(self, project):
```

Comment:
QUESTION: Was trying to find a way more inline with new functional test to get this setup cutdown, using `profiles_config_update` but didn't receive any real luck or should we directly override the `dbt_profile_target`

Reply:
profiles_config_update is for dbt_project yml, but you're handing it target params. Use the target fixture and this should be alright.

Reply:
will play around with that method again was trying to either trigger it via that but hit errors so tried the config update, or using a new conftest option which felt bloated as we wouldn't use it anywhere else.